lib/ts/controllers/s-expandable-control.ts

```diff
@@ -145,12 +147,13 @@
             // attribute; for checkable controls this also means setting the `is-collapsed` class
             this.element.setAttribute("aria-expanded", this.isCollapsed() ? "false" : "true");
             if (this.isCheckable) {
-                var cc = this.controlledCollapsible;
+                var cc = this.controlledCollapsibles;
                 if (cc) {
```

Comment:
`cc` will always be truthy here:

```suggestion
                if (cc.length) {
```